src/CustomMarshalers/CustomMarshalers.sln

```diff
@@ -0,0 +1,32 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.136
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E9D942E3-D65A-44F8-835E-831066E41788}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{88142883-B4AF-4C3B-9B8F-4501407A824E}"
```

Comment:
Empty ref directory? Why have it then?

Reply:
Also, any way to test this stuff?

Reply:
I'll go remove it. I was planning on adding a ref assembly but I don't know if that's needed.

Reply:
I was going to add tests in coreclr since they need to have a native component.